db/migrate/20231009105044_add_delete_at_to_roles.rb

```diff
@@ -0,0 +1,7 @@
+class AddDeleteAtToRoles < ActiveRecord::Migration[6.1]
+  def change
+    change_table(:roles) do |t|
+      t.date :delete_on
+    end
+  end
+end
```

Comment:
there might be existing roles with deleted_at date in the future.
we need to migrate them, don't we?
how the handle these? after_destroy hooks are probably executed already.

Reply:
Ja, müssen wir unbedingt migrieren. Sonst gelten ab dem Release plötzlich alle existierenden Rollen mit zukünftigem Löschdatum wieder als bereits heute gelöscht.

Reply:
Habe migration entsprechend ergänzt